students.rb

```diff
@@ -0,0 +1,10 @@
+# Named Students to differentiate it from the already existing Student class
+class Students
+  attr_accessor :name, :classroom, :rentals
+
```

Comment:
- [ ] _Student_ object belongs to the _Classroom_. Let's endeavor to establish a belong to association accordingly like:
```ruby
class Student < Person
  attr_reader :classroom
  .
  .
   def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
end
```

Reply:
Kindly establish the belong to association as suggested by the CR. 😉